libs/community/langchain_community/document_loaders/blob_loaders/cloud_blob_loader.py

```diff
@@ -0,0 +1,294 @@
+"""Use to load blobs from the local file system."""
+import contextlib
+import mimetypes
+import tempfile
+from io import BufferedReader, BytesIO
+from pathlib import Path
+from typing import (
+    TYPE_CHECKING,
+    Callable,
+    Generator,
+    Iterable,
+    Iterator,
+    Optional,
+    Sequence,
+    TypeVar,
+    Union,
+)
+from urllib.parse import urlparse
+
+if TYPE_CHECKING:
+    from cloudpathlib import AnyPath
+
+from langchain_community.document_loaders.blob_loaders.schema import (
+    Blob,
+    BlobLoader,
+)
+
+T = TypeVar("T")
+
+
+class CloudBlob(Blob):
```

Comment:
Could you mark this as private so users, only code to the `Blob` interface and not to the `CloudBlob`?

```suggestion
class _CloudBlob(Blob):
```

Reply:
done